ui/app/templates/vault/cluster/policies/index.hbs

```diff
@@ -91,52 +91,39 @@
               </LinkTo>
             </div>
             <div class="column has-text-right">
-              <PopupMenu name="policy-nav">
-                <nav class="menu">
-                  <ul class="menu-list">
-                    {{#if item.updatePath.isPending}}
-                      <li class="action">
-                        <LoadingDropdownOption />
-                      </li>
-                      <li class="action">
-                        <LoadingDropdownOption />
-                      </li>
-                    {{else}}
-                      {{#if item.canRead}}
-                        <li class="action">
-                          <LinkTo
-                            @route="vault.cluster.policy.show"
-                            @models={{array this.policyType item.id}}
-                            data-test-policy-link="show"
-                          >
-                            Details
-                          </LinkTo>
-                        </li>
-                      {{/if}}
-                      {{#if item.canEdit}}
-                        <li class="action">
-                          <LinkTo
-                            @route="vault.cluster.policy.edit"
-                            @models={{array this.policyType item.id}}
-                            data-test-policy-link="edit"
-                          >
-                            Edit
-                          </LinkTo>
-                        </li>
-                      {{/if}}
-                      {{#if item.canDelete}}
-                        <ConfirmAction
-                          @isInDropdown={{true}}
-                          @buttonText="Delete"
-                          @confirmTitle="Delete this policy?"
-                          @confirmMessage="This will permanently delete this policy and may affect access to some data"
-                          @onConfirmAction={{action "deletePolicy" item}}
-                        />
-                      {{/if}}
-                    {{/if}}
-                  </ul>
-                </nav>
-              </PopupMenu>
+              <Hds::Dropdown @isInline={{true}} as |dd|>
```

Comment:
Good question! That's the default! So I left it out to shorten the args, but I can add it back for clarity

Reply:
I added it since `@isInline` could inherit styling from the parent, so decided to explicitly define while we're still getting accustom to using the `Hds::Dropdown`